it/controllers/RegistrationConfirmationControllerISpec.scala

```diff
@@ -25,7 +28,7 @@ class RegistrationConfirmationControllerISpec extends RegSummaryISpecHelper {
                 .copy(submittedOn = Some(submittedDate))
               val createdSubscriptionAndAmountProducedGlobally = CreatedSubscriptionAndAmountProducedGlobally(
                 Subscription.generate(userAnswers, RosmWithUtr("0000001611", rosmRegistration)), HowManyLitresGlobally.Large)
-              given
+              `given`
```

Comment:
https://github.com/hmrc/soft-drinks-industry-levy-account-frontend/pull/42/files#diff-68745ac2beddef22ff9dc87cd38ed0af98c037ecd16f7aa67c4c5a1c35b689d4R37

Emmanuel used "build" here, I think it may be cleaner than using the backticks. Can likely update this using a global search and replace